srcpkgs/python3-ansible-lint/template

```diff
@@ -19,13 +19,13 @@ maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
 license="GPL-3.0-only"
 homepage="https://github.com/ansible-community/ansible-lint"
 distfiles="${PYPI_SITE}/a/${pkgname/python3-/}/${pkgname/python3-/}-${version}.tar.gz"
-checksum=b539bc22d13e6de0cc2e25758e1d28b2bc01561a414ae37ceda3708b5a2a79ed
+checksum=34491240d3e5405eee26471f3a00ab0d54724eb60a359e2424864a544e6097d4
 # Tests are currently broken for ansible on python 3.10
 # See https://github.com/ansible/ansible/issues/74658
 # and https://github.com/ansible/ansible/issues/74660
 make_check=no
 
 post_patch() {
 	# scm versioning is broken in this release
```

Comment:
This comment is now outdated.

Reply:
I think this comment was outdated before I adopted them as well, no?

Would you be happy with just skipping a comment? Alternatively, "Explicitly set version as it can't be discovered from the scm automatically"